pkg/types/contract_writer.go

```diff
@@ -51,6 +51,7 @@ const (
 	Unknown TransactionStatus = iota
 	Pending
 	Unconfirmed
+	Confirmed
```

Comment:
Why do we need both Unconfirmed and Confirmed?

Reply:
Yeah if `Unconfirmed` means mined but not finalized, then I don't see what `Confirmed` is for :thinking: